<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_agregar</class>
 <widget class="QDialog" name="d_agregar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="b_agregar">
     <property name="text">
      <string>AGREGAR</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTreeWidget" name="t_lista_a">
     <column>
      <property name="text">
       <string>Archivos</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="b_cancelar">
     <property name="text">
      <string>CANCELAR</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
